system/datetime-plugin/.gitignore

```diff
@@ -0,0 +1,3 @@
+/node_modules
+/build
+/src/w3
```

Comment:
need to add `wrap` and `.polywrap` here. The `src/wrap/` folder got added to git because of this, can you remove it?